### ISSUE TITLE
In the ceph package error: non-const getNamespaceDef() called on aliased member. 

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#7660]
With Fossies the number of ceph package gave (over 2000 times) the message:
```
error: non-const getNamespaceDef() called on aliased member. Please report as a bug.
```
This problem differs from #7624 as the messages are given in the part: "Searching for member function documentation..."
An attempt has been made to make the problem a bit smaller:  [example.tar.gz](https://github.com/doxygen/doxygen/files/4365646/example.tar.gz)
(hopefully the fix for this problem will also fix, most, of the other problems with the ceph package)

The original source code can be found at: https://github.com/ceph/ceph/archive/v15.1.1.tar.gz
The used Doxyfile to reproduce the original ceph problem: [Doxyfile_ceph.tar.gz](https://github.com/doxygen/doxygen/files/4365648/Doxyfile_ceph.tar.gz)

**Comment by albert-github** [comment]
The commit e4070d58bb8d8ee8fe63c33480732b963f33c23f  fixed the problem.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
